Models/CustomerPayment.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+
+namespace BangazonAPI.Models
+{
+    public class CustomerPayment
+    {
+        [Key]
+        public int CustomerPaymentId { get; set; }
+
+        [Required]
+        public int CardNumber { get; set; }
+
+        [StringLength(3,ErrorMessage = "CCV code is three numbers"),Required]
+        public string CcvCode { get; set; }
+
+        [StringLength(10, MinimumLength = 4,ErrorMessage = "Expiration Date is four to ten numbers"),Required]
+        public string ExpirationDate { get; set;}
+        //Foreign Keys
+        [Required]
+        public int CustomerId { get; set; }
```

Comment:
I can't find anything in the tickets that says we'll need to show the the actual `Customer` when we use this model. 

If you include something that's not in the ERD, please reference the ticket number so we can cross-reference.

Reply:
(this was in reference to line 22 but I clicked the wrong line number, sorry!)

Reply:
removed lines as directed